css/style.css

```diff
@@ -9,6 +9,35 @@
 }
 
 /* BEGIN Header Menu */
+.menu-bar-container {
+  width: 100%;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background: #3e3c3c;
+}
+
+.portfolio-menu {
+  display: none;
+  justify-content: flex-end;
+  margin-right: 10rem;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 20px;
+  text-align: center;
+  align-items: center;
+}
+
+.desktop-menu {
+  display: none;
+}
+
+#hamburger {
+  cursor: pointer;
+}
```

Comment:
- [ ] **Kindly As requested by the previous reviewer**,  I think it would be better to follow the Figma design, 🙏 and as suggested by the previous reviewer, 
    - Please make sure you remove the horizontal scroll you have, due to this horizontal overflow, after I click on the hamburger icon and want to close the mobile menu, I have to scroll horizontally before I will see the mobile menu close, 🙏 

you may try to style  ```body``` in your CSS file to increase the use of mobile devices, By styling the body, you can ensure that your website is readable and accessible on a range of devices, including smartphones, tablets, and desktops. 👍 👍 

Kindly try to follow this [link](https://www.freecodecamp.org/news/responsive-web-design-modern-website-code-for-beginners/) for more